.github/workflows/examples-telink.yaml

```diff
@@ -99,6 +99,15 @@ jobs:
                     out/telink-tlsr9518adk80d-ota-requestor/zephyr/zephyr.elf \
                     /tmp/bloat_reports/
 
+            - name: Build example Telink Thermostat App
```

Comment:
did the dockerfile version not need changing in this file?

Reply:
No